### ISSUE TITLE
Waiter timeout error while deploying

### ISSUE THREAD

**Issue by chobbs** [influxdata/google-deployment-manager-influxdb-enterprise#2]
In the [setup-meta.sh], the command for creating the admin database user is falling causing the waiter not to succeed and timeout:

https://github.com/influxdata/google-deployment-manager-influxdb-enterprise/blob/7cd667487ede50eec2f66752067c0fadc9f59604/simple/setup-meta.sh#L74

**Comment by chobbs** [comment]
Added fix to starc branch -- @gunnaraasen

**Comment by fareeda0** [comment]
Hey, I tried running with the fix applied to the starc branch and tried to deploy via gcloud cli but the waiter is still timing out. Any ideas?

**Comment by chobbs** [comment]
@fareeda0 - Thanks, let me  investigate

**Comment by chobbs** [comment]
Introduced a regression in the starc branch -- please try the **ga-packer** branch:   @fareeda0

**Comment by fareeda0** [comment]
@chobbs Thanks! Tried the same with the ga-packer branch and no change. The waiter still times out.

**Comment by chobbs** [comment]
@fareeda0 - Did you pull the latest branch? I pushed an update to the ga-packer branch yesterday that contained the fix. Can you send me the error you're getting?

**Comment by fareeda0** [comment]
> @fareeda0 - Did you pull the latest branch? I pushed an update to the ga-packer branch yesterday that contained the fix. Can you send me the error you're getting?

Yep, I pulled the ga-packer branch, with the latest commit ‘curl fix’. 

What I did was cd into the simple directory, and run the gcloud command as per the .MD instructions.

On GCP, I can see the deployment running in deployment manager - then exactly at the 10 minute mark, the waiter times out and the automatic rollback kicks in and deletes the deployment.

I don’t have the exact error, it’s along the lines of ‘waiter timed out’.  However, it always fails at the 600s mark which is the timeout set for waiter.

I’ve tried increasing the timeout and again it’ll fail at whatever it’s been set to (i.e. 900s)

Thanks!

**Comment by fareeda0** [comment]
@chobbs Just wondering if there's any updates on this. Keen to try out, please let me know if there's anything I can do to help.

Thanks!

**Comment by chobbs** [comment]
@fareeda0 - We have not be able to reproduce your error, however we did recently find a bug in our **byol** template. Can you tell me if you were entering in your own license key?

**Comment by chobbs** [comment]
Hi @fareeda0 -- wanted to reach out again and see if your still having the timeout. We've made some updates to the deployment templates that may resolve your issue.

**Comment by fareeda0** [comment]
Hey @chobbs that's great news!

To answer your previous question, yep I was providing my own license key from the 14 day trial. 

I am very keen to try it out again, however my trial key has since expired and I wasn't able to evaluate the product.

**Comment by fareeda0** [comment]
@chobbs -- just tried again from the Web UI, and it timed out again.

The only changes I made were changing the region to europe-west1 and changing the size of the data node SSDs from 250GB to 50GB

See the screenshot below.

[https://imgur.com/a/q1eSt0t](url)

**Comment by chobbs** [comment]
Thanks @fareeda0 - lets go ahead and open a support case. If you don't mind sending an email to support@influxdata.com - include my name name and  the details from this here.
